src/components/SecondaryPanes/Expressions.js

```diff
@@ -9,17 +9,19 @@ import { bindActionCreators } from "redux";
 import actions from "../../actions";
 import { getExpressions, getLoadedObjects, getPause } from "../../selectors";
 import { getValue } from "../../utils/expressions";
-
+import * as parser from "../../workers/parser";
```

Comment:
lets just import the one function we need `import { hasX`

Reply:
sure